content/open-source/oci/what-is-the-oci.md

```diff
@@ -0,0 +1,111 @@
+---
+title: "What is the Open Container Initiative?"
+type: "article"
+description: " ... (fill out)"
```

Comment:
I think you forgot to set the description here! this shows up in the list page of articles under the "open container" menu.